snippets/pad.md

```diff
@@ -0,0 +1,16 @@
+### pad
+
+Pads `string` on the left and right side if it's shorter than `length`.
+String is *NOT* truncated on both sides if it exceeds length.
+
+```js
+const pad = (string, length = 8, char = ' ') =>
+  string.padStart(string.length + (length - string.length) / 2, char).padEnd(length, char);
```

Comment:
`string.length/2` should be `Math.floor(string.length/2)` just to be safe against people looking for fun.

Reply:
Could you give me an example what "fun" I might look for? Just being curious...